config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  root 'pages#home'
-  get 'pages/home'
+  root 'pages#homepage'
+  get 'pages/homepage', to: 'pages#homepage'
```

Comment:
I think it would be nicer if you stick to the URIs that we discussed in class:

```
/
/attend
/incompany
/sponsor
```

You can remove the `/pages` part from it 😉